tests/nlp/conftest.py

```diff
@@ -21,15 +21,15 @@
 from deepchecks.nlp.text_data import TextData
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope='function')
 def text_classification_dataset_mock():
     """Mock for a text classification dataset"""
     return TextData(raw_text=['I think therefore I am', 'I am therefore I think', 'I am'],
                     label=[0, 0, 1],
                     task_type='text_classification')
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope='function')
```

Comment:
Will this not slow our tests?

Reply:
probably a little bit/
but those fixtures are mutated within tests
we do not what to share mutable fixtures across all tests as it will make tests not idempotent